appveyor.yml

```diff
@@ -0,0 +1,70 @@
+version: 0.5.61.{build}
+image: Visual Studio 2015
+
+
+environment:
+  matrix:
+    - PlatformToolset: v120_xp
+    - PlatformToolset: v140_xp
+
+platform:
+    - x64
+    - Win32
+
+configuration:
+    - Unicode Debug
+    - Unicode Release
+
+
+install:
+    - if "%platform%"=="x64" set archi=amd64
+    - if "%platform%"=="Win32" set archi=x86
+    - call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" %archi%
+
+build:
+    parallel: true                  # enable MSBuild parallel builds
+    verbosity: minimal
+
+build_script:
+    - cd "%APPVEYOR_BUILD_FOLDER%"
+    - msbuild FingerText.vcxproj /m /p:configuration="%configuration%" /p:platform="%platform%" /p:PlatformToolset="%PlatformToolset%" 
+
+after_build:
+    - cd "%APPVEYOR_BUILD_FOLDER%"
+    - ps: >-
+
+        if ($env:PLATFORM -eq "x64") {
+            #Push-AppveyorArtifact "$($env:PLATFORM)\$($env:CONFIGURATION)\FingerText.dll" -FileName FingerText.dll
+        }
+
+        if ($env:PLATFORM -eq "Win32" ) {
+            #Push-AppveyorArtifact "$($env:CONFIGURATION)\FingerText.dll" -FileName FingerText.dll
+        }
+
+        if ($($env:APPVEYOR_REPO_TAG) -eq "true" -and $env:CONFIGURATION -eq "Unicode Release" -and $env:PLATFORMTOOLSET -eq "v120_xp") {
+            if($env:PLATFORM -eq "x64"){
+            $ZipFileName = "FingerText_$($env:APPVEYOR_REPO_TAG_NAME)_x64.zip"
+            7z a $ZipFileName $env:PLATFORM\$env:CONFIGURATION\FingerText.dll
+            }
+            if($env:PLATFORM -eq "Win32"){
+            $ZipFileName = "FingerText_$($env:APPVEYOR_REPO_TAG_NAME)_x86.zip"
+            7z a $ZipFileName $env:CONFIGURATION\FingerText.dll 
+            }
+        }
+
+artifacts:
+  - path: FingerText_*.zip
+    name: releases
+
+deploy:
+    provider: GitHub
+    auth_token:
+        secure: !!TODO, see https://www.appveyor.com/docs/deployment/github/#provider-settings!!
```

Comment:
token needed to enable github deployment

Reply:
It looks like you have a 64bit build almost completed except for this. Can you finish this out? This is a must-have plugin for Notepad++!

Reply:
@sigzero This is something which just could be added by the project owner.